canonical/ubuntu.com
# Landing refresh

_Continuation of https://github.com/canonical/ubuntu.com/pull/13755_
## Done

- Applied rebrand as per [design](https://www.figma.com/file/Le4TTzyOFbcbqTPJ1tVoZT/U.com---Home-(rebranding)?node-id=0%3A1&mode=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13803.demos.haus/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against design
- Check against doc and prod see that no functionality has changed 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8630
